distribution_list/distribution_list.py

```diff
@@ -292,8 +296,12 @@ def get_complex_distribution_list_ids(self, cr, uid, ids, context=None):
                         else:
                             result_ids .append(val[main_object])
                 if alternative_object:
```

Comment:
Is-it possible to split the computation of main_object and alternative_object into 2 sub methods to improve the readability and the modularity?